deployment/nixops/prometheus.nix

```diff
@@ -82,7 +93,7 @@ in
 
     networking.firewall = {
       enable = true;
-      allowedTCPPorts = [ 22 3000 ];
+      allowedTCPPorts = [ 22 80 ];
```

Comment:
Ditto? (I'm not being snarky, I honestly don't know. Often worth assuming future readers will be suitably ignorant.)

It also gets used further down.

Reply:
Hmm, pulling out into a let block inside prometheus.nix makes sense, I don't think it makes sense to pull out of prometheus.nix though.